ysql_test.go

```diff
@@ -533,7 +533,7 @@ func getExpiration(t testing.TB, db *ysql, username string) time.Time {
 		return time.Time{} // No expiration
 	}
 
-	exp, err := time.Parse("2006-01-02 15:04:05Z07", rawExp) //time.RFC3339
+	exp, err := time.Parse(time.RFC3339, rawExp) //time.RFC3339
```

Comment:
Just curious - what does this do since it does not seem related to the failure? nit: remove the commented out part

Reply:
The data-type of `valuntil` column of the `pg_catalog.pg_user` table is `abstime` in PG 11 and `timestamp with time zone` in PG15, due to which we need different time format to parse it for the respective PG versions.